<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionViajes</class>
 <widget class="QMainWindow" name="GestionViajes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gesti√≥n de Viajes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tabla_viajes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>760</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_actualizar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualizar Viaje</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_eliminar">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>450</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar Viaje</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
</ui>
